pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/map/MapIndexType.java

```diff
@@ -55,6 +55,7 @@ public class MapIndexType extends AbstractIndexType<MapIndexConfig, MapIndexRead
       Collections.singletonList(V1Constants.Indexes.MAP_INDEX_FILE_EXTENSION);
   private static final String MAP_INDEX_CREATOR_CLASS_NAME = "mapIndexCreatorClassName";
   private static final String MAP_INDEX_READER_CLASS_NAME = "mapIndexReaderClassName";
+  private static final String MAP_INDEX_MUTABLE_CLASS_NAME = "mapIndexMutableClassName";
```

Comment:
call it `mutableMapIndex` instead of `mapIndexMutable`?

Reply:
Sure. Let me change it, I was just trying to have consistent prefix but I think we can have MUTABLE_MAP_INDEX_CLASS_NAME = "mutableMapIndexClassName"